### PR TITLE
Sys 3828 upgrade substrate version

### PR DESCRIPTION
**Pull request by ivan-cholakov**
## Proposed changes

- Upgraded from v0.9.36 to v1.0.0. 
- Updated dependencies versions. 
- Change the processed_events type on genesis from EthEventId to a tuple of  H256. 
- Recalculated benchmarks

[Screencast from 22-02-24 11:07:24.webm](https://github.com/AventusProtocolFoundation/avn-parachain/assets/7727203/99ebbb6b-e932-448e-af56-1e8d3d70e912)

Tested locally and on increment chain

## Type of change/Merge

🚨What type of change is this PR? </br>
_Put an `x` in the boxes that apply_

- [x] Release 
  - [x] Increase versions 
  - [x] Baseline tests passed  
  - Release type:
    - [ ] Major release 
    - [x] Minor release 
    - [ ] Patch release 

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x] You describe the purpose of the PR, e.g.:
  - What does it do?
  - Highlight what important points reviewers should know about;
  - Indicates if there is something left for follow-up PRs.
- [ ] Documentation updated
- [x] Business logic tested successfully
- [ ] Verify First, Write Last: In Substrate development, it is important that you always ensure preconditions are met and return errors at the beginning. After these checks have completed, then you may begin the function's computation.

### CODE REVIEW

File: pallets/token-manager/Cargo.toml [hunk 1]
```diff
@@ -13,55 +13,59 @@ rust-version = { workspace = true }
...
 [features]
 default = ["std"]
 std = [
 	"frame-benchmarking?/std",
-	"serde",
+	"scale-info/std",
 	"codec/std",
 	"sp-std/std",
+    "sp-io/std",
+    "sp-core/std",
```
Comment (thadouk) [inline pallets/token-manager/Cargo.toml:58]: Indentation

File: runtime/avn/src/lib.rs [hunk 2]
```diff
@@ -69,7 +68,7 @@ use pallet_avn::sr25519::AuthorityId as AvnId;
 
 pub use pallet_avn_proxy::{Event as AvnProxyEvent, ProvableProxy};
 use pallet_avn_transaction_payment::AvnCurrencyAdapter;
-pub use pallet_ethereum_events::Event as EthEvent;
+// pub use pallet_ethereum_events::Event as EthEvent;
```
Comment (thadouk) [inline runtime/avn/src/lib.rs:71]:
```suggestion
```

**Review by thadouk — APPROVED** [review]

File: pallets/validators-manager/src/tests/mock.rs [hunk 3]
```diff
@@ -425,7 +432,10 @@ impl ExtBuilder {
     pub fn as_externality(self) -> sp_io::TestExternalities {
         let mut ext = sp_io::TestExternalities::from(self.storage);
         // Events do not get emitted on block 0, so we increment the block here
-        ext.execute_with(|| frame_system::Pallet::<TestRuntime>::set_block_number(1u32.into()));
+        ext.execute_with(|| {
+            Timestamp::set_timestamp(1);
```
[the remark this answers is not in the document]
Comment (ivan-cholakov) [reply pallets/validators-manager/src/tests/mock.rs:436]: this was needed for the issue with the logs when running the tests - to satisfy the error with timestamp 0

File: pallets/parachain-staking/Cargo.toml [hunk 4]
```diff
@@ -57,13 +62,16 @@ std = [
 	"serde",
 	"sp-runtime/std",
 	"sp-std/std",
+    "sp-io/std",
```
[the remark this answers is not in the document]
Comment (ivan-cholakov) [reply pallets/parachain-staking/Cargo.toml:65]: ran `cargo fmt`, shouldn't have indentation issues, this is probably visual bug

File: pallets/ethereum-events/src/lib.rs [hunk 5]
```diff
@@ -381,8 +386,11 @@ pub mod pallet {
 
             EventChallengePeriod::<T>::put(self.event_challenge_period);
 
-            for (key, value) in self.processed_events.iter() {
-                ProcessedEvents::<T>::insert(key, value);
+            for (key, key2, value) in self.processed_events.iter() {
+                ProcessedEvents::<T>::insert(
+                    EthEventId { signature: key.clone(), transaction_hash: key2.clone() },
+                    value,
+                );
```
[the remark this answers is not in the document]
Comment (ivan-cholakov) [reply pallets/ethereum-events/src/lib.rs:393]: can't do this as it needs to be cloned anyway

**Review by nahuseyoum — APPROVED** [review]

**Review by fluorostani — APPROVED** [review]

**Pull request merged**